Build is failing when project gets loaded into IntelliJ

Build is failing when you previously loaded it into IntelliJ

Spotless is failing because it analyses files in the `.idea` folder that is created by IntelliJ

Output of `gradlew clean build`:

```
Starting a Gradle Daemon (subsequent builds will be faster)                                                                                           
> Task :spotlessXmlCheck FAILED                                                                                                                       
                                                                                                                                                      
FAILURE: Build failed with an exception.                                                                                                              
                                                                                                                                                      
* What went wrong:                                                                                                                                    
Execution failed for task ':spotlessXmlCheck'.                                                                                                        
> The following files had format violations:                                                                                                          
      .idea\codeStyles\codeStyleConfig.xml                                                                                                            
          @@ -1,5 +1,5 @@                                                                                                                             
          -<component name="ProjectCodeStyleConfiguration">\r\n                                                                                       
          -  <state>\r\n                                                                                                                              
          -    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />\r\n                                                                     
          -  </state>\r\n                                                                                                                             
          +<component name="ProjectCodeStyleConfiguration">\n                                                                                         
          +  <state>\n                                                                                                                                
          +    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Default" />\n                                                                       
          +  </state>\n                                                                                                                               
           </component>                                                                                                                               
      .idea\codeStyles\Project.xml                                                                                                                    
          @@ -1,29 +1,29 @@                                                                                                                           
          -<component name="ProjectCodeStyleConfiguration">\r\n                                                                                       
          -  <code_scheme name="Project" version="173">\r\n                                                                                           
          -    <JavaCodeStyleSettings>\r\n                                                                                                            
          -      <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="256" />\r\n                                                                
          -      <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="256" />\r\n                                                                
          -      <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">\r\n                                                                                 
          -        <value />\r\n                                                                                                                      
          -      </option>\r\n                                                                                                                        
          -    </JavaCodeStyleSettings>\r\n                                                                                                           
          -    <JetCodeStyleSettings>\r\n                                                                                                             
          -      <option name="PACKAGES_TO_USE_STAR_IMPORTS">\r\n                                                                                     
          -        <value>\r\n                                                                                                                        
          -          <package name="java.util" alias="false" withSubpackages="false" />\r\n                                                           
          -          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />\r\n                                            
          -          <package name="io.ktor" alias="false" withSubpackages="true" />\r\n                                                              
          -        </value>\r\n                                                                                                                       
          -      </option>\r\n                                                                                                                        
          -      <option name="PACKAGES_IMPORT_LAYOUT">\r\n                                                                                           
          -        <value>\r\n                                                                                                                        
          -          <package name="" alias="false" withSubpackages="true" />\r\n                                                                     
          -          <package name="java" alias="false" withSubpackages="true" />\r\n                                                                 
          -          <package name="javax" alias="false" withSubpackages="true" />\r\n                                                                
          -          <package name="kotlin" alias="false" withSubpackages="true" />\r\n                                                               
          -          <package name="" alias="true" withSubpackages="true" />\r\n                                                                      
          -        </value>\r\n                                                                                                                       
          -      </option>\r\n                                                                                                                        
          -    </JetCodeStyleSettings>\r\n                                                                                                            
          -  </code_scheme>\r\n                                                                                                                       
          +<component name="ProjectCodeStyleConfiguration">\n                                                                                         
          +  <code_scheme name="Project" version="173">\n                                                                                             
          +    <JavaCodeStyleSettings>\n                                                                                                              
          +      <option name="CLASS_COUNT_TO_USE_IMPORT_ON_DEMAND" value="256" />\n                                                                  
          +      <option name="NAMES_COUNT_TO_USE_IMPORT_ON_DEMAND" value="256" />\n                                                                  
          +      <option name="PACKAGES_TO_USE_IMPORT_ON_DEMAND">\n                                                                                   
          +        <value />\n                                                                                                                        
          +      </option>\n                                                                                                                          
          +    </JavaCodeStyleSettings>\n                                                                                                             
      ... (20 more lines that didn't fit)                                                                                                             
  Violations also present in:                                                                                                                         
      .idea\compiler.xml                                                                                                                              
      .idea\gradle.xml                                                                                                                                
      .idea\jarRepositories.xml                                                                                                                       
      .idea\misc.xml                                                                                                                                  
      .idea\vcs.xml                                                                                                                                   
  Run 'gradlew.bat :spotlessApply' to fix these violations.                                                                                           
                                                                                                                                                      
* Try:                                                                                                                                                
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.  
                                                                                                                                                      
* Get more help at https://help.gradle.org                                                                                                            
                                                                                                                                                      
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.                                                           
Use '--warning-mode all' to show the individual deprecation warnings.                                                                                 
See https://docs.gradle.org/6.7/userguide/command_line_interface.html#sec:command_line_warnings                                                       
                                                                                                                                                      
BUILD FAILED in 1m 9s                                                                                                                                 
20 actionable tasks: 20 executed                                                                                                                      
```

--- a/build.gradle
+++ b/build.gradle
@@ -79,7 +79,7 @@ spotless {
 		eclipse().configFile "misc/eclipse_formatting_nowrap.xml"
 	}
 	format 'xml', {
-		targetExclude 'build/test-results'
+		targetExclude 'build/test-results', fileTree('.idea')
 		target '**/*.xml', '**/*.nuspec'
 	}
 }

